infrastructure/aws/trigger_blame_rebuild.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import boto3
+from datetime import datetime, timedelta
+import sys
+import os.path
+
+# Usage: trigger_blame_rebuild.py <mozsearch-repo> <config-repo> <config-input> <branch> <channel>
+#  e.g.: trigger_blame_rebuild.py https://github.com/mozsearch/mozsearch https://github.com/mozsearch/mozsearch-mozilla config1.json master release
+
+def trigger(mozsearch_repo, config_repo, config_input, branch, channel):
+    ec2 = boto3.resource('ec2')
+    client = boto3.client('ec2')
+
+    running = ec2.instances.filter(Filters=[{'Name': 'tag-key', 'Values': ['blame-builder']},
+                                           {'Name': 'tag:channel', 'Values': [channel]},
+                                           {'Name': 'instance-state-name', 'Values': ['running']}])
+    for instance in running:
+        print("Terminating existing running indexer %s for channel %s" % (instance.instance_id, channel))
+        instance.terminate()
+
+    user_data = '''#!/usr/bin/env bash
+
+cd ~ubuntu
+sudo -i -u ubuntu ./update.sh "{branch}" "{mozsearch_repo}" "{config_repo}"
+sudo -i -u ubuntu mozsearch/infrastructure/aws/main.sh rebuild-blame.sh 168 "{branch}" "{channel}" config "{config_input}"
+'''.format(branch=branch, channel=channel, mozsearch_repo=mozsearch_repo, config_repo=config_repo, config_input=config_input)
+
+    block_devices = []
+
+    images = client.describe_images(Filters=[{'Name': 'name', 'Values': ['indexer-18.04']}])
```

Comment:
s/18.04/20.04/ (and maybe we should delete or rename the 18.04 AMI's now?  I left them around for faster rollback.)

Reply:
Should be safe to delete them now, I think. We've had a few days of running with the new ones and no issues seem to have cropped up.

Reply:
I deleted the 18.04 AMI's so now we're down to just the web+indexer 20.04 AMI's.